### PR TITLE
fix: version message, add test for version

### PR DESCRIPTION
**Pull request by axif0**
### Contributor checklist

- [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch

---

### Description

- Fix version message
- Add tests for Version Check. with command `pytest tests
` 

### Related issue

- #190 
- #188

### CODE REVIEW

**Review by mhmohona — APPROVED** [review]
Hello @axif0, looks like you have addressed the review in https://github.com/scribe-org/Scribe-Data/pull/170#issuecomment-2282873796. Thanks for your great work! ✨ 

@andrewtavis, do you think for each command we should have separate test file like Asif has proposed?

**Comment by andrewtavis** [comment]
Yes, I think that the way that @axif0 has it set up works well :) If the other tests in #170 could be split and added to `tests/cli` like the one here that would be great!

**Comment by andrewtavis** [comment]
Thanks for the work here both of you! 😊

File: tests/cli/test_version.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+    @patch("scribe_data.cli.version.get_local_version", return_value="1.0.0")
+    @patch(
+        "scribe_data.cli.version.get_latest_version", return_value="Scribe-Data v1.0.1"
+    )
+    def test_get_version_message_update_available(
+        self, mock_latest_version, mock_local_version
+    ):
+        """
+        Tests the scenario where a newer version is available, suggesting an update.
```
Comment (andrewtavis) [inline tests/cli/test_version.py:77]: Very minor thing here that is more a personal thing for me, @axif0, I added the periods here as Python standards state that comments that are their own line should have a period at the end (and should also start with a capitalized work - they're full sentences). For inline comments it's best to not capitalize the first word and not end in a period :)

For next time! 😊

**Review by andrewtavis — APPROVED** [review]
Really appreciate the changes here, and especially adding in the tests, @axif0! Great work! Thanks for all your contributions today :) :)

**Pull request merged**